docker/composable-sandbox.docker

```diff
@@ -0,0 +1,52 @@
+FROM debian:11-slim as builder
```

Comment:
why not nixos with flakes? :)))

Reply:
innovation! we are web 3.0

Reply:
imho, using nix to build a docker image is better because it will have a smaller size. but a usual Dockerfile can support more people from our team.